src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +23,23 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws IOException {
+        Set<Character> treeSet = new TreeSet<>();
+        StringBuilder stringWithUniqueCharacters = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            while (bufferedReader.ready()) {
+                int letter = bufferedReader.read();
+                if (Character.isAlphabetic(letter)) {
```

Comment:
```suggestion
                if (Character.isLetter(letter)) {
```
check out the difference)